backend/.env.docker

```
@@ -0,0 +1,10 @@
+MONGODB_URI="mongodb://mongo-service/wanderlust"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)